frontend/src/pages/projects/screens/projects/tableData.tsx

```diff
@@ -4,28 +4,54 @@ import { getProjectCreationTime, getProjectDisplayName } from '~/pages/projects/
 
 export const columns: SortableData<ProjectKind>[] = [
   {
-    field: 'name',
-    label: 'Name',
-    sortable: (a, b) => getProjectDisplayName(a).localeCompare(getProjectDisplayName(b)),
+    field: 'project',
+    label: 'Project',
+    sortable: false,
+    colSpan: 2,
+    hasRightBorder: true,
+    width: 50,
   },
   {
-    field: 'notebooks',
-    label: 'Workbench',
+    field: 'Workbenches',
+    label: 'Workbenches',
     sortable: false,
+    colSpan: 2,
+    hasRightBorder: true,
+    width: 40,
   },
   {
-    field: 'notebook-status',
-    label: 'Status',
+    field: 'kebab',
+    label: '',
     sortable: false,
+    rowSpan: 2,
+    width: 10,
+  },
+];
+export const subColumns: SortableData<ProjectKind>[] = [
+  {
+    field: 'name',
+    label: 'Name',
+    sortable: (a, b) => getProjectDisplayName(a).localeCompare(getProjectDisplayName(b)),
```

Comment:
These do not materialize in the table as sortable configurations...

Reply:
Fixed table to allow sorting on sub-headings